src/web/modules/statistics/statistics.http.js

```diff
@@ -91,25 +94,35 @@ const byServices = async function byServices(req, res, next) {
       compareDate.add(1, 'month')
     } while (compareDate.isBefore(toDate))
 
-    let serviceNameMap = {}
+    let serviceDataMap = {}
     services.forEach((service) => {
       Object.assign(
-        serviceNameMap,
+        serviceDataMap,
         service.gateway_account_ids.reduce((aggregate, accountId) => {
           aggregate[accountId] = {
             service: service.service_name && service.service_name.en,
-            organisation: service.merchant_details && service.merchant_details.name
+            organisation: service.merchant_details && service.merchant_details.name,
+            sector: service.sector,
+            went_live_date: service.went_live_date,
+            internal: service.internal,
+            archived: service.archived,
           }
           return aggregate
         }, {})
       )
     })
 
     const liveGatewayAccounts = accounts
-      .filter((account) => account.type === 'live')
+      .filter((account) => account.type === 'live'
+        && serviceDataMap[account.gateway_account_id]
+        && !serviceDataMap[account.gateway_account_id].internal
+        && !serviceDataMap[account.gateway_account_id].archived)
```

Comment:
As you already did in `map` that follows, this could read a bit clearer with assigning the service data first
```suggestion
      .filter((account) => {
        const serviceData = serviceDataMap[account.gateway_account_id]
        return account.type === 'live'
          && serviceData
          && !serviceData.internal
          && !serviceData.archived
      })
```